docs-src/concepts/what-is-worker-versioning.md

```diff
@@ -8,127 +8,118 @@ tags:
-to change the type signature of an Activity Definition while creating a new incompatible build ID
-for a Worker, you can do so without concern that the Activity will fail to execute on some other
-Worker with an incompatible definition. The same principle applies to Child Workflows. It's
-important to keep in mind that "publicly facing" workflows on a versioned Task Queue should not
-change their signature, as this defeats the purpose of workflow-launching clients being oblivious
-to changes in the Workflow Definition. If you need to change the signature of a Workflow, you should
-use a different Workflow Type or an entirely new Task Queue.
-
-Note that if you schedule an Activity or a Child Workflow on _different_ Task Queue from the one
-the Workflow is running on, they will be not be assigned any particular version. Meaning if the
-target queue is versioned, they will run on the latest default, and if it is unversioned, they will
-operate as they normally would have without this feature.
-
-Continue-as-new from a Workflow on a versioned Task Queue will, by default, start the continued
-Workflow on the same compatible set as the original Workflow. Continuing-as-new onto a different
-task queue will not assign any particular version. You may also optionally indicate that the
-continued workflow should start using the latest default version of the Task Queue.
+Worker Versioning simplifies the process of deploying changes to [Workflow Definitions](/workflows/#workflow-definition). It does this by allowing you to define sets of versions that are compatible with each other, and then assigning a Build ID to the code that defines a Worker. The Temporal Server uses the Build ID to determine which versions of a Workflow Definition a Worker can process.
+
+We recommend that you read about Workflow Definitions before proceeding, because Workflow Versioning is largely concerned with helping to manage nondeterministic changes to those definitions.
+
+Worker Versioning helps manage nondeterministic changes by providing a convenient way to ensure that [Workers](#workers) with different Workflow and Activity Definitions operating on the same [Task Queue](#task-queue) don't attempt to process [Workflow Tasks](#workflow-task) and [Activity Tasks](#activity-task-execution) that they can't successfully process, according to sets of versions associated with that Task Queue that you've defined.
+
+Accomplish this goal by assigning a Build ID (a free-form string) to the code that defines a Worker, and specifying which Build IDs are compatible with each other by updating the version sets associated with the Task Queue, stored by the Temporal Server.
+
+### When and why should you use Worker Versioning
+
+The main reason to use this feature is to deploy incompatible changes to short-lived [Workflows](/workflows). When you use this feature on Task Queues, the Workflow starter doesn't have to know about the introduction of new versions.
```

Comment:
This one needs to stay "On Task Queues using this feature" I think. Both phrasings are accurate in some sense, but I think the important thing here is that Task Queues which use the feature act with different behavior